src/main/java/org/mafagafogigante/dungeon/schema/rules/SpecificIdJsonRule.java

```diff
@@ -0,0 +1,31 @@
+package org.mafagafogigante.dungeon.schema.rules;
+
+import org.mafagafogigante.dungeon.game.Id;
+
+import com.eclipsesource.json.JsonValue;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+class SpecificIdJsonRule extends IdJsonRule {
+
+  private final Set<Id> ids = new HashSet<>();
+
+  SpecificIdJsonRule(Collection<String> stringIds) {
+    for (String stringId : stringIds) {
```

Comment:
If this gets changed to Collection<Id>, prefer using addAll(), even if the JVM does not perform any optimization, at least it is shorter and easier to read.